packages/components/src/Form/Fields/Field/FieldInline.tsx

```diff
@@ -53,21 +54,29 @@ const FieldInlineLayout: FC<FieldInlinePropsInternal> = ({
   required,
   validationMessage,
 }) => {
+  const describedbyId = `${id}-description`
+
+  const inputWithAriaDescribed = isValidElement(children)
+    ? React.cloneElement(children, {
+        'aria-describedby': describedbyId,
+      })
+    : children
+
   return (
-    <label className={className} htmlFor={id}>
-      <InputArea>{children}</InputArea>
-      <Label as="span">
+    <div className={className}>
+      <InputArea>{inputWithAriaDescribed}</InputArea>
+      <Label htmlFor={id}>
         <Truncate>{label}</Truncate>
         {required && <RequiredStar />}
       </Label>
       {detail && <FieldDetail>{detail}</FieldDetail>}
-      <MessageArea id={`${id}-describedby`}>
-        {description && <FieldDescription>{description}</FieldDescription>}
-        {validationMessage ? (
-          <ValidationMessage {...validationMessage} />
-        ) : null}
+      <MessageArea>
```

Comment:
ARIA id needs to be assigned here to keep the "validationMessage" tied to the input for screen-reader announcement.

```suggestion
      <MessageArea id={`${id}-describedby`}>
```